### PR TITLE
Add support for Laravel 9

### PR DESCRIPTION
**Pull request by jclarkkoala**
# What

Updates composer dependencies to work with Laravel 9.

# Why

Laravel 8 is approaching EOL on January 24, 2023.

# CC 🐨

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,12 +25,39 @@ jobs:
...
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-testing-php8-laravel8-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-composer-
+      - name: Set testbench dependency
+        run: composer require orchestra/testbench "^6"
+#      - name: Composer
+#        run: composer install
```
Comment (walterbm) [inline .github/workflows/ci.yml]: @jclarkkoala does this workflow need to run `composer install` or was this just a test?

**Review by walterbm — APPROVED** [review]
Looks great! Thank you for adding Laravel 8 and 9 workflows — glad this didn't require any real code changes!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jclarkkoala) [reply .github/workflows/ci.yml]: Composer installation is triggered by the `composer require orchestra/testbench ^6` above it.

**Comment by jclarkkoala** [comment]
@walterbm 
Re-requesting a review for the change to the CI testing step.
The updated testing step takes advantage of GitHub Actions' matrix declaration to build combinations of PHP versions and Laravel versions, and runs a test job using each combination.

The test covers -
Laravel ^8 on the latest minor release for PHP versions 7.4, 8.0, 8.1
Laravel ^9 on the latest minor release for PHP versions ^8.0, ^8.1, ^8.2

Information on GitHub Actions `matrix`:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,52 +6,40 @@ on:
       - main
 
 jobs:
-  testing-8:
+  testing-php-laravel-combinations:
+    name: Tests
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
```
Comment (walterbm) [inline .github/workflows/ci.yml:14]: Great use of a matrix!

**Review by walterbm — APPROVED** [review]

**Pull request merged**